src/pybind/mgr/cephadm/upgrade.py

```diff
@@ -174,7 +174,7 @@ def upgrade_status(self) -> orchestrator.UpgradeStatusSpec:
             # accessing self.upgrade_info_str will throw an exception if it
             # has not been set in _do_upgrade yet
             try:
-                r.message = self.upgrade_info_str
+                r.message = self.upgrade_info_str  # type: ignore
```

Comment:
Why did you need to add `type: ignore` to this line?

Reply:
For whatever reason, mypy suddenly noticed (and this was before your mypy version bump) that the variable didn't have a type
```
cephadm/upgrade.py: note: In member "upgrade_status" of class "CephadmUpgrade":
cephadm/upgrade.py:177: error: Cannot determine type of "upgrade_info_str"
```
I've just set the type in the class initialization now instead of the ignore, but still don't understand what changes I made here that made mypy become aware of it suddenly